src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started his work!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped his work!");

```

Comment:
Similarly, the message 'Truck stopped his work!' could be more informative. You should clarify that it is a `Truck` that has stopped working, for instance: 'Truck has stopped working.'